demo/src/screens/incubatorScreens/IncubatorExpandableOverlayScreen.tsx

```diff
@@ -68,7 +68,7 @@ export default class TextFieldScreen extends Component {
 
   renderPickerContent = () => {
     return (
-      <View bg-white br20 padding-s3 paddingB-60>
+      <View bg-$backgroundDefault br20 padding-s3 paddingB-60>
```

Comment:
Is this bg still needed?

Reply:
yes, try to remove it :)

Reply:
I did, please describe the problem, or maybe I missed something in the reproduction

Reply:
remove it and open the second picker in the expandableOverlay screen, you'll see that the dialog has no bg

Reply:
🤦‍♂️ 
Totally ignored that example...